src/audio-manager.ts

```diff
@@ -0,0 +1,226 @@
+import {_audioContext} from './audio-listener.js';
+
+const preloadedBuffers: {[key: string]: [Promise<AudioBuffer>, number]} = {};
```

Comment:
Use a `Map<>` here, it's a little better than object literals for those kind of stuff